lighthouse-core/audits/accessibility/aria-valid-attr-value.js

```diff
@@ -33,6 +33,8 @@ class ARIAValidAttr extends AxeAudit {
       category: 'Accessibility',
       name: 'aria-valid-attr-value',
       description: 'Element aria-* attributes have valid values',
+      helpText: 'Screen readers may not be able to read elements with invalid ARIA ' +
```

Comment:
Suggest rewording to something like "Assistive technology, such as a screen reader, will not be able to interpret ARIA attributes with invalid values"